build-logic/convention/src/main/kotlin/GravatarComposeConventionPlugin.kt

```diff
@@ -0,0 +1,61 @@
+import com.android.build.gradle.LibraryExtension
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.kotlin.dsl.apply
+import org.gradle.kotlin.dsl.configure
+import org.gradle.kotlin.dsl.withType
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+class GravatarComposeConventionPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        with(target) {
+            apply(plugin = "io.github.takahirom.roborazzi")
+
+            extensions.configure<LibraryExtension> {
+                buildFeatures {
+                    compose = true
+                }
+                composeOptions {
+                    kotlinCompilerExtensionVersion = "1.5.15"
```

Comment:
Minor, but is it possible to extract this to the toml file?

Reply:
It is. It doesn't work as nicely as in the `.gradle` files, but you can access it. I've pushed the changes.

Reply:
Looks good, thanks :)